README.md

```diff
@@ -38,43 +42,58 @@ This is something that you get for free just by adding the following dependency
 </dependency>
 ```
 
-Then you can inject the client in your cloud simply by:
+Then you can inject the client in your code simply by:
 
 ```java
 @Autowire
 private DiscoveryClient discoveryClient;
 ```
 
-If for any reason you need to disable the `DiscoveryClient` you can simply set the following property:
+If for any reason you need to disable the `DiscoveryClient` you can simply set the following property in `application
+.properties`:
 
 ```
 spring.cloud.kubernetes.discovery.enabled=false
 ```
 
-Some spring cloud components use the `DiscoveryClient` in order obtain info about the local service instance. For this to work you need to align the service name with `spring.application.name`.
+// TODO: make clearer with an example and details on how to align service and application name
```

Comment:
No. According to this [thread](See: https://stackoverflow.com/questions/4823468/comments-in-markdown), here is the syntax to be used

```
[//]: # "TODO: make clearer with an example and details on how to align service and application name"
```